cloud_sql/mysql_instance_read_pool/main.tf

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# [START cloud_sql_mysql_read_pool]
+
+resource "google_sql_database_instance" "primary" {
+  name             = "mysql-primary"
+  database_version = "MYSQL_8_0"
```

Comment:
Does this sample work with the most recent MySQL version? Extended support for 8.0 starts [in less than a year](https://cloud.google.com/sql/docs/db-versions); if the latest version supports this functionality, please use the latest vesion: 
```suggestion
  database_version = "MYSQL_8_4"
```